backend/de.metas.adempiere.adempiere/base/src/main/java-legacy/org/compiere/model/PO.java

```diff
@@ -1951,6 +1965,22 @@ private boolean loadColumn(final int index)
 	 * @return true if loaded
 	 */
 	protected final boolean load(final HashMap<String, String> hmIn)
+	{
+		final PerformanceMonitoringService service = SpringContextHolder.instance.getBeanOr(
```

Comment:
discussed with @metas-ts the issues from these lines. pls clarify with him tomorrow.

Reply:
briefly:
* move to PerformanceMonitoringService service to POServicesFacade
* consider converting `PerformanceMonitoringService.Metadata.builder()...build()` to constant